### ISSUE TITLE
doesn't seems to work

### ISSUE THREAD

**Issue by Grummfy** [spatie/laravel-cors#18]
Hello,
I'm using laravel 5.6.*, and I don't see any header added to my response.

The config file
```
<?php
return [
	'cors_profile' => Spatie\Cors\CorsProfile\DefaultProfile::class,
	'default_profile' => [
		'allow_origins' => [
			env('APP_URL'),
		],
		'allow_methods' => [
			'POST',
			'GET',
		],
		'allow_headers' => [
			'Content-Type',
			'X-Auth-Token',
			'Origin',
			'Authorization',
		],
		'max_age' => 60 * 60 * 24,
	],
];
```

The cors have been added to the midleware, and on the route list I see it.
Did I miss something?

**Comment by freekmurze** [comment]
Add a  `dd` in the cors profile class to verify if you installed it correctly.

**Comment by Grummfy** [comment]
inded I pu dd in setRequest, in isAllowed, and in several other methods and nothing appends.

**Comment by freekmurze** [comment]
Could you add a `dd` in the middleware class this package provides?

**Comment by Grummfy** [comment]
It give the correct output for the boot

**Comment by freekmurze** [comment]
The tests seem to indicate that the package is working correctly.

Could you try installing it in a vanilla Laravel app?

**Comment by Grummfy** [comment]
it's working well, just bad context (not cors context) AND I was assuming that the cors header would be shown ;)

**Comment by notflip** [comment]
What is a bad context? I seem to be having the same issue

**Comment by samuelting** [comment]
I have the same issue on Access-Control-Allow-Origin header not appearing.

I found that I did missed some points as stated in the README.
"...For more infomation about CORS, see the Mozilla CORS documentation. https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS"

In the Mozilla documentation, following the section "Examples of access control scenarios", it mentioned
that 
"The use of the Origin header and of Access-Control-Allow-Origin show the access control protocol in its simplest use. In this case, the server responds with Access-Control-Allow-Origin: *, which means that the resource can be accessed by any domain."

You probably missed adding "Origin" and this library does not consider your request as CORS request. Therefore it does not show "Access-Control-Allow-Origin" header in your response.

Read the doc carefully before proceeding...
